verilog/formatting/tree_unwrapper.cc

```diff
@@ -1923,6 +1923,15 @@ void TreeUnwrapper::Visit(const verible::SyntaxTreeLeaf& leaf) {
   // Should be equivalent to AdvanceNextUnfilteredToken().
   AddTokenToCurrentUnwrappedLine();
 
+  // Make sure that newly extended unwrapped line has origin symbol
+  // FIXME: Pretty sure that this should be handled in an other way,
```

Comment:
CC @fangism who might have a good guidance of how _should be handled in an other way_ can be

Reply:
Generally, the origin pointer is a little trickier when partitions reshape and boundaries move.  Its intent was to help with context-sensitive formatting decisions like alignment and diagnostics.  

This case is interesting because a `;` by itself is a [kNullDeclaration](https://cs.opensource.google/verible/verible/+/master:verilog/parser/verilog.y;drc=61a6c055aad09454994f353e091dfb9b9cfb9a21;l=1312) symbol.
Maybe the partition construction should treat this symbol as a valid class-item for the purpose of alignment, but perhaps ignore it because it doesn't match the structure of other declarations.  Alignment-wise, it can be treated as an ignored row.